apps/website-25/src/lib/routes.ts

```diff
@@ -0,0 +1,30 @@
+export type BluedotRoute = {
+  /**
+   * Title of the page, by convention this is the title that appears in the
+   * <title /> tag (e.g. "About us" in "About us | BlueDot Impact")
+   */
+  title: string;
+  /**
+   * Relative url of the route (e.g. /about, not https://bluedot.org/about)
+   */
+  url: string;
+};
+
+export const ROUTES = {
+  home: {
+    title: 'BlueDot Impact',
```

Comment:
the designs have this as "Bluedot.org" but I'm also thinking "Home" might be best? will sync with PM tomorrow to confirm

```suggestion
    title: 'Home',
```

Reply:
Cool, just let me know 👍. I would like to try and keep the tab title the same if possible, which is why I called it "BlueDot Impact" instead of "Bluedot.org" (sorry I forgot to flag that). So if changing it to "Home" I think it would make sense to change the title from:
`<title>{CURRENT_ROUTE.title} | Industry-leading free AI courses and career support</title>`
to:
`<title>{CURRENT_ROUTE.title} | BlueDot Impact</title>`